core/shared/src/main/scala/scalaz/zio/UIO.scala

```diff
@@ -0,0 +1,534 @@
+package scalaz.zio
+
+import scalaz.zio.Exit.Cause
+import scalaz.zio.internal.{ Executor, Platform }
+
+/*
+ * PHASE 1 DONE
+ * PHASE 2 DONE
+ * PHASE 3 DONE
+ * PHASE 4 DONE
+ */
+
+/*
+ * EXTENDS ZIOFunctions
+ */
+object UIO {
+
+  type UioE = Nothing
+  type UioR = Any
```

Comment:
As in the other cases, I'd delete:

1. Temp comments
2. Scaladoc (link to ZIO Scaladoc)
3. Type aliases `UioE` and `UioR`
4. Because `UIO` doesn't use environment, delete the environmental functions (`provide`, `access`, `accessM`, `environment`)